bower.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "ovh-manager-webfont",
     "description": "Managers front",
-    "version": "1.1.0",
+    "version": "1.2.0",
```

Comment:
This doesn't need to be done manually and shouldn't

Reply:
This has been undone